rpc_util.go

```diff
@@ -365,6 +366,36 @@ func (o MaxRecvMsgSizeCallOption) before(c *callInfo) error {
 }
 func (o MaxRecvMsgSizeCallOption) after(*callInfo, *csAttempt) {}
 
+// CallAuthority returns a CallOption which sets the authority to override the
+// `:authority` pseudoheader in a RPC. If this is set, the RPC will use only
+// this authority.
+//
+// # Experimental
+//
+// Notice: This API is EXPERIMENTAL and may be changed or removed in a
+// later release.
+func CallAuthority(auth string) CallOption {
+	return AuthorityOverrideCallOption{Authority: auth}
+}
+
+// AuthorityOverrideCallOption is a CallOption which sets the authority that the
```

Comment:
Same here

AuthorityOverrideCallOption is a CallOption that indicates the HTTP/2 :authority header value to use for the call.